dev/com.ibm.websphere.appserver.thirdparty.eclipselink.2.7/resources/org/eclipse/persistence/exceptions/i18n/SDOExceptionResource.nlsprops

```diff
@@ -0,0 +1,147 @@
+
+45014=Could not define type.  Types can only be defined for DataObjects with Type set to commonj.sdo.Type
+
+45015=Could not define type with a null name.
+
+45016=A modified object in the changesummary XML is missing a ref attribute or the value is not specified.
+
+45017=An error occurred processing xpath [{0}] .
+
+45018=Adding a duplicate entry for the complex single setting [{1}] into a sequence at position [{0}] is not supported.
+
+45019=Adding an attribute property [{0}] to a sequence is not supported.
+
+45020=No sequence found for path [{0}].
+
+45021=An error occurred attempting to instantiate a Sequence object with a null dataObject instance field.
+
+45022=No sequence is supported for property [{0}].
+
+45023=Could not set property of type with uri [{0}] and name [{1}] to value of class [{2}]
+
+45024=A conversion error occurred.
+
+45025=Could not find property at index [{0}]
+
+45026=Cannot perform operation [{0}] on null parameter.
+
+45027=Cannot find class for type with uri [{0}] and name [{1}].
+
+45028=A type can not be set to open and datatype.  Error with type with uri [{0}] and name [{1}].
+
+45029=Invalid index [{0}] passed to method [{1}].
+
+45030=An error occurred invoking the constructor with a String argument on the class [{0}].
+
+45031=Cannot set invalid target type [{0}] on property [{1}] because target type.dataType is true.
+
+45032=An XMLMarshalException occurred for uri [{1}] and local name [{2}].  Exception: [{0}]
+
+45033=An error occurred generating types. The XML Schema component with name [{1}] and namespace URI [{0}] is referenced but never defined.  An import or include for namespace URI[{0}] might be missing from the XML Schema.
+
+45034=The value of the options parameter must be a DataObject of Type with uri [{0}] and name [{1}].
+
+45035=The value that corresponds to the "type" property must be a Type object.
+
+45036=A global property corresponding to the XML node being loaded was not found.
+
+45037=The prefix [{0}] is used but not declared in the XML schema.
+
+45038=Cannot perform operation on property [{0}] because it is not reachable from the path [{1}]. The path is invalid, or one or more Data Objects on the path are null.
+
+45039=An error occurred accessing the externalizableDelegator field [{0}] on the DataObject.
+
+45040=Cannot perform operation [{0}] with null parameter [{1}].
+
+45041=The value [{0}] of class [{1}] is invalid for property [{2}] of type [{3}].
+
+45100=An error occurred attempting to return the requested SDOHelperContext.  In an active WebLogic instance the application name is required for helper context cache lookups.  The application name could not be determined because {0} lookup failed.
+
+45101=An error occurred attempting to return the requested SDOHelperContext.  In an active WebLogic instance the application name is required for helper context cache lookups.  The application name could not be determined because {0} could not be invoked reflectively on {1}.
```

Comment:
What are 0 and 1? Files, classes, directories, etc? Fill in blah with the correct noun that describes 0 and 1, i.e. the 0 class, and the 1 method: "The application name could not be determined because the {0} _blah_ could not be invoked reflectively on the {1} _blah_.".